ntbs-service/Pages/Notifications/NotificationErrorSummary.cshtml

```diff
@@ -0,0 +1,41 @@
+@model ntbs_service.Pages_Notifications.NotificationModelBase
+
+@{
+    Layout = "../Shared/_NotificationLayout.cshtml";
+    Dictionary<string, List<string>> ErrorDictionary = (Dictionary<string, List<string>>) ViewData["ErrorDictionary"];
+}
+
+@{
+    var ErrorToPageMapping = new Dictionary<string, string> {
+        {"PatientDetails", "Patient Details"},
+        {"ClinicalDetails", "Clinical Details"},
+        {"Episode", "Hospital Details"},
+        {"PatientTBHistory", "Previous History"}
+    };
+
+    var ErrorToUrlMapping = new Dictionary<string, string> {
+        {"PatientDetails", "Patient"},
+        {"ClinicalDetails", "ClinicalDetails"},
+        {"Episode", "Episode"},
+        {"PatientTBHistory", "PreviousHistory"},
+    };
+}
+
+<nhs-error-summary title-text="You cannot proceed until the following issues are resolved" title-id="error-summary-title"
+role="alert" tabindex="-1" data-module="error-summary">
+    @foreach (var errorMapping in ErrorToPageMapping) 
+    {
+        @if (ErrorDictionary.ContainsKey(errorMapping.Key) && ErrorDictionary[errorMapping.Key].Count > 0) 
+        {
+            <li>
+                @{string page_url = $"/Notifications/Edit/{ErrorToUrlMapping[errorMapping.Key]}?id={Model.NotificationId}&isBeingSubmitted={Model.Notification.ShouldValidateFull}";}

```

Comment:
I think we can simplify this to `isBeingSubmitted=true` :)

Reply:
indeed :)